app/experimenter/templates/experiments/detail_ship.html

```diff
@@ -41,33 +41,35 @@ <h5 class="modal-title" id="normandyModalLabel">Normandy Recipe</h5>
         <div class="modal-body">
           <p>Enter the following information into Delivery Console exactly as shown:</p>
           <pre class="bg-light json prettyprint">{{ experiment.normandy_recipe_json }}</pre>
-          <p>When the recipe is saved in Delivery Console, copy the Recipe ID below and hit save.</p>
-        </div>
 
-        <div class="modal-footer">
 
```

Comment:
here is where I bidded adieu the modal-footer... which really wasn't loving the embedded form... and me trying to make rows and columns in the form...

Reply:
BYEEEEEEEEE
![giphy](https://user-images.githubusercontent.com/119884/61903624-e8a5a800-aef2-11e9-9e30-49ba825a0b4e.gif)